### ISSUE TITLE
Use cases to showcase value of the tool 

### ISSUE THREAD

**Issue by patwater** [argo-marketplace/LA-LocalGeo-CrossWalk#25]
Several folks in LA have expressed interest in this tool though would help to have a couple prototype use cases to showcase its value. One thought is to slice and dice the [Hack for LA Civic Art project](http://okfnlabs.org/projects/) by city council district and other aggregations to showcase the value. 

Another idea would be to slice up the impact of the [new transit oriented development law done by our friends at Policy Club ](https://github.com/policyclub/zoning-metadata) by other geospatial aggregations. 

CC @gauravcusp @lingyielia @vr00n 

# How to submit an issue?

- [x] Have you checked previously open/closed issues to see if your question has already been asked & addressed? 
  - If there is an open issue similar to your issue. Comment on that open issue with additional detail indicating that you too are having the same issue. This allows the project sponsor to prioritize incoming issues.
- [x] Have you provided a clear description of your issue so that the project sponsor is able to answer all your questions with as few back-n-forths as possible?
- [x] (Please acknowledge by checking this box) If a screenshot or visual helps better communicate some complex idea, have you provided one?. `Draw.io` and `lucidchart` are some excellent and quick drawing tools.
- [x] (Please acknowledge by checking this box) When an issue is resolved, please post the steps to resolution or clearly describe the resolution before marking the issue closed. 
   - This will help future `SOLVERS` read through how your issue was solved and may help them solve their issue quicker.
- [x] Have you been kind, courteous, and considerate in your questions? 
- [x] Have you considered using `Stackedit.io`, a handy tool to quickly write in markdown?

**Comment by gauravcusp** [comment]
Thats awesome @patwater ! We can discuss about it over our call on 14th. Until then @lingyielia  and I will go through the links you shared.

**Comment by vr00n** [comment]
Thanks for prompting this discussion. 

I think the most compelling use-cases emerge when you have some city/county-wide data (like energy/water consumption, homelessness levels, diabetes, obesity, crime, motor vehicle crashes or 311 complaints). 

Using this API - you can slice/dice these datasets across all these various **local geographies** to quickly identify potential opportunities for inter-agency collaborations or civic action.

Hypothetical scenario: 

Say youth crime was particularly high in a specific `congressional district` or `state senate district`- you could develop a quick app to send a mail or postcard using www.lob.com to that area's congressman or state senator with these analytics.

OR

you can integrate with a dataset like NYC's facilities explorer https://capitalplanning.nyc.gov/map/facilities#10/40.7128/-74.0807 to identify inter-agency collaboration.

**Comment by patwater** [comment]
Yes I agree. The two links I suggested were citywide art projects and citywide zoning data respectively
